README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
@alarimer: There are a few steps to do this, but trying to best describe. Assuming you did this change through GitHub, I'm detailing configuration steps as well.

First, you need to clone your fork of this repo to a local folder in your computer using git:

```shell
git clone git@github.com:alarimer/grunt-contrib-qunit.git
```

Now you can checkout to the branch you have the changes from this PR.

```shell
git checkout patch-1
```

Then, you can go the the project's folder, install the project dependencies using __npm__:

```shell
cd grunt-contrib-qunit
npm install
```

If you have __node__ installed in your machine you should have __npm__ installed in your path as well.

Now you can run grunt to verify everything is all set. You can have grunt installed globally or use the local version as the following:

```shell
./node_modules/.bin/grunt
```

Now you can edit that file, save it,and run grunt again, just like the command above.

It should generate the README.md file properly.

You can check you're ready to commit the changes:

```shell
git status
```

After that, add the files to be staged for a new commit:

```
git add <file>
```

If git status shows you only the files you want to commit (the modified file and README.md), you can add them with a shortcut, instead of the command above:

```
git add .
```

Now you commit the changes:

```
git commit -m "fixup! message"
```

and push the commit to the respective branch (_patch-1_) remote repo (_origin_) on GitHub:

```
git push origin patch-1
```

This Pull Request page should show your new commit here. Remember to ping @Arkni back so they can check your fixes and merge the Pull Request.

---

I'm sorry if this was too detailed, but I know it's a struggle when we need to change from a version control system to another thing. Still, I'm glad you're contributing to this project. Thanks!